packages/components/checkbox/src/setup.ts

```diff
@@ -0,0 +1,111 @@
+import { computed, ComputedRef, inject, Ref, ref, watch, getCurrentInstance, SetupContext } from 'vue'
+import { checkboxGroupInjectionKey, subjectInjectKey } from './checkbox'
+import { isArray, hasSlot } from '@idux/cdk/utils'
+import type { CheckboxProps, RawBindings } from './types'
+import { useAttrs } from '@idux/components/core/utils/useAttrs'
```

Comment:
```suggestion
import { useAttrs } from '@idux/components/core/utils'
```

在 utils/index.ts 里 export 一下